### PR TITLE
added configure conan

### PR DESCRIPTION
**Pull request by mcb5637**

### CODE REVIEW

**Review by Clueliss — CHANGES_REQUESTED** [review]
Why are the actions in seperate directories?

**Comment by mcb5637** [comment]
Because it needs to be called action.yml, otherwise GitHub complains that it can not find it.
See https://github.com/dice-group/dice-sparse-map/actions/runs/9317805578/job/25648861616

**Comment by Clueliss** [comment]
> Because it needs to be called action.yml, otherwise GitHub complains that it can not find it. See https://github.com/dice-group/dice-sparse-map/actions/runs/9317805578/job/25648861616

I think thats because they need to be in the workflows directory, no idea why but on every other action we have here it works that way

**Comment by mcb5637** [comment]
These are actions, just single steps in a workflow.
If they are in the workflow folder, that gives a different error.

**Review by Clueliss — APPROVED** [review]

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -82,26 +70,12 @@ jobs:
         uses: rui314/setup-mold@v1
 
       - name: Install conan
-        env:
-          CC: ${{ steps.install_cc.outputs.cc }}
-          CXX: ${{ steps.install_cc.outputs.cxx }}
+        uses: dice-group/cpp-conan-release-reusable-workflow/.github/actions/configure_conan
```
[the remark this answers is not in the document]
Comment (mcb5637) [reply .github/workflows/publish-release.yml:73]: this repo, main branch
i tested with the previous commit, which was pointing at this branch

**Review by Clueliss — APPROVED** [review]

**Pull request merged**